integration_tests/models/marts/metrics.yml

```diff
@@ -26,4 +26,4 @@ metrics:
 
     meta: 
       team: "Finance"
-      refresh_rate: "Bob's weekly run"
\ No newline at end of file
+      refresh_rate: "Bob's weekly run"
```

Comment:
was this a net new addition? not sure why this is flagged as a change here

Reply:
Not net new no - seems to be showing as a deletion followed by replacement. Sorry, may have got my branch in a bit of a tizz - I can try and resolve if you need, but don't think it has any difference from what's in main. Let me know, and thanks for your support in contributing - very happy to be involved!